src/components/LaunchDetailsPage/mocks/launchDetails.ts

```diff
@@ -0,0 +1,18 @@
+export const launchDetails = {
+  name: 'Long March 4B (?) | SJ-6 Group 06 (?)',
+  status: {
+    name: 'Go for Launch',
+  },
+  net: '2022-05-07T07:45:00Z',
+  launch_service_provider: {
```

Comment:
rename this field with camelCase

Reply:
changed, but this name comes from backend